Accordion.js

```diff
@@ -71,20 +74,23 @@ export default class Accordion extends Component {
       }
     });
 
+    const Touchable = this.props.touchableComponent;
+
     return (
       <View {...viewProps}>
         {this.props.sections.map((section, key) =>
           <View key={key}>
-            <TouchableHighlight
+            <Touchable
               onPress={() => this._toggleSection(key)}
               underlayColor={this.props.underlayColor}
+              {...this.props.touchableProps}
```

Comment:
Cool, I guess JSX spreads work differently than regular object spreads then. Fix the README typo and we're good to go 👍